client/client-test/Description.spec.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Description from '../components/Description';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+ 
+configure({ adapter: new Adapter() });
+ 
+describe('Renders Components', () => {
+  let component;
+  let props = {
```

Comment:
Interesting way to pass props. I just keep a sampleData file that I import into my test suite. But I like this way because you can see exactly what goes in.